<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentsDialog</class>
 <widget class="QWidget" name="ComponentsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Components</string>
  </property>
  <property name="windowIcon">
   <iconset>
    :/zxtune.png
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="componentsTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pluginsTab">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="pluginsTree">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="backendsTab">
      <attribute name="title">
       <string>Sound backends</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="backendsTree">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="providersTab">
      <attribute name="title">
       <string>IO providers</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="providersTree">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string notr="true">Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ComponentsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
